cms/plugins/text/templates/cms/plugins/widgets/ckeditor.html

```diff
@@ -5,6 +5,17 @@
 var editPluginPopupCallbacks = {};
 var CK_BRIDGE = null;
 
+CKEDITOR.on("instanceReady", function(event){
+    if(window.self !== window.parent){
+        //inside iframe
+        try{
+            var url = window.location.href.replace(/\/$/, ''),
```

Comment:
I think it's better to not fetch the plugin id by parsing the url. I think it's better to somehow pass it into the template context.